internal/pkg/testenv/testenv.go

```diff
@@ -131,6 +137,10 @@ func (env *TestEnv) WriteFile(t *testing.T, fileName string, content string) {
 	defer f.Close()
 	_, err = f.WriteString(content)
 	assert.NoError(t, err)
+	err = os.Chtimes(env.GetPath(fileName),
+		time.Date(2006, time.February, 1, 3, 4, 5, 0, time.UTC),
+		time.Date(2006, time.February, 1, 3, 4, 5, 0, time.UTC))
```

Comment:
What is the significance of this date?

Reply:
Nothing, or it was the first example on stack overflow.  And its 1.2.3.4.5.6